licenses.yaml

```diff
@@ -233,6 +233,30 @@ notice: |
 
 ---
 
+name: JavaBeans Activation Framework API JAR
```

Comment:
Are these only run by CI? Unless these jars end up in the source binary packaging when building the distributions I don't believe they need to be included here.

Reply:
They're part of the distribution now. Here's a diff of the distribution:
```
Only in before/extensions/druid-hdfs-storage: api-util-1.0.0-M20.jar
Only in after/extensions/druid-hdfs-storage: api-util-1.0.3.jar
Only in before/extensions/druid-kafka-extraction-namespace: kafka-clients-2.1.0.jar
Only in after/extensions/druid-kafka-extraction-namespace: kafka-clients-2.1.1.jar
Only in before/extensions/druid-kerberos: api-util-1.0.0-M20.jar
Only in after/extensions/druid-kerberos: api-util-1.0.3.jar
Only in after/extensions/druid-kinesis-indexing-service: jackson-annotations-2.10.1.jar
Only in before/extensions/druid-kinesis-indexing-service: jackson-annotations-2.9.10.jar
Only in after/extensions/druid-kinesis-indexing-service: jackson-core-2.10.1.jar
Only in before/extensions/druid-kinesis-indexing-service: jackson-core-2.9.10.jar
Only in after/extensions/druid-kinesis-indexing-service: jackson-databind-2.10.1.jar
Only in before/extensions/druid-kinesis-indexing-service: jackson-databind-2.9.10.jar
Only in before/extensions/druid-orc-extensions: protobuf-java-3.1.0.jar
Only in after/extensions/druid-orc-extensions: protobuf-java-3.11.0.jar
Only in after/extensions/druid-protobuf-extensions: error_prone_annotations-2.3.2.jar
Only in before/extensions/druid-protobuf-extensions: gson-2.7.jar
Only in after/extensions/druid-protobuf-extensions: gson-2.8.6.jar
Only in after/extensions/druid-protobuf-extensions: protobuf-java-3.11.0.jar
Only in before/extensions/druid-protobuf-extensions: protobuf-java-3.2.0.jar
Only in after/extensions/druid-protobuf-extensions: protobuf-java-util-3.11.0.jar
Only in before/extensions/druid-protobuf-extensions: protobuf-java-util-3.2.0.jar
Only in after/lib: jackson-annotations-2.10.1.jar
Only in before/lib: jackson-annotations-2.9.10.jar
Only in after/lib: jackson-core-2.10.1.jar
Only in before/lib: jackson-core-2.9.10.jar
Only in after/lib: jackson-databind-2.10.1.jar
Only in before/lib: jackson-databind-2.9.10.jar
Only in after/lib: jackson-dataformat-cbor-2.10.1.jar
Only in before/lib: jackson-dataformat-cbor-2.9.10.jar
Only in after/lib: jackson-dataformat-smile-2.10.1.jar
Only in before/lib: jackson-dataformat-smile-2.9.10.jar
Only in after/lib: jackson-datatype-guava-2.10.1.jar
Only in before/lib: jackson-datatype-guava-2.9.10.jar
Only in after/lib: jackson-datatype-joda-2.10.1.jar
Only in before/lib: jackson-datatype-joda-2.9.10.jar
Only in after/lib: jackson-jaxrs-base-2.10.1.jar
Only in before/lib: jackson-jaxrs-base-2.9.10.jar
Only in after/lib: jackson-jaxrs-json-provider-2.10.1.jar
Only in before/lib: jackson-jaxrs-json-provider-2.9.10.jar
Only in after/lib: jackson-jaxrs-smile-provider-2.10.1.jar
Only in before/lib: jackson-jaxrs-smile-provider-2.9.10.jar
Only in after/lib: jackson-module-guice-2.10.1.jar
Only in before/lib: jackson-module-guice-2.9.10.jar
Only in after/lib: jackson-module-jaxb-annotations-2.10.1.jar
Only in before/lib: jackson-module-jaxb-annotations-2.9.10.jar
Only in after/lib: jakarta.activation-api-1.2.1.jar
Only in after/lib: jakarta.xml.bind-api-2.3.2.jar
Only in before/lib: protobuf-java-3.1.0.jar
Only in after/lib: protobuf-java-3.11.0.jar
Only in after/licenses/bin: jakarta.activation-api.EDL1
```